### PR TITLE
Add target for Wio LTE.

### PR DESCRIPTION
**Pull request by matsujirushi**

### CODE REVIEW

File: Devices/WioLTE/Device.h [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+#pragma once
+
+#define STM32F405xx 1
+
+#include <STM32F4.h>
+
+#define DEVICE_TARGET		STM32F4
+#define DEVICE_NAME			"WioLTE"
+#define DEVICE_MANUFACTURER	"Seeed"
```
Comment (Arke64) [inline Devices/WioLTE/Device.h]: We should use the full name, Seeed Studio.

File: Devices/WioLTE/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+## What's Wio LTE?
+
+![](https://github.com/SeeedDocument/Wio_Tracker_LTE/raw/master/img/wio_tracker_lte_1.jpg)
+
+The Wio LTE is well suited for outdoor projects where the device can connect to the GPS satellites and provide a real-time location of the item it is attached to. The LTE provides a wide bandwidth which allows much faster interaction between the user and device. Some example projects could be a bicycle sharing service, tracking pets or livestock, locating a vehicle, or even keeping track of a child.
+
+
+## Links
+
+* The [Wiki]() of Wio LE - 4G, Cat. 1
```
Comment (Arke64) [inline Devices/WioLTE/README.md]: This link should be filled out

File: README.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ Some targets and devices are maintained by GHI Electronics while others are main
 - G30
 - G80
 - USBizi
+- [WiFiMCU](http://www.wifimcu.com/)
```
Comment (Arke64) [inline README.md]: These two links shouldn't be under the list officially maintained by us. I'd create a separate section after the one talking about official GHI devices for community devices and put these two there.

**Comment by Arke64** [comment]
Thanks for the contribution! There are just a few small things that need to be addressed before we can accept the PR

File: Devices/WioLTE/Device.h (hunk 1 above)
Comment (matsujirushi) [reply Devices/WioLTE/Device.h]: 'Seeed' is correct.
https://www.seeedstudio.com/about_us.html

File: Devices/WioLTE/README.md (hunk 2 above)
Comment (matsujirushi) [reply Devices/WioLTE/README.md]: Thanks.
I'll fix it.

File: README.md (hunk 3 above)
Comment (matsujirushi) [reply README.md]: I see.
I'll rollback README.md.

**Comment by matsujirushi** [comment]
Fixed. Please review.

**Comment by Arke64** [comment]
Thanks, looks good. I just pushed a commit that fixed a few tabs that should be spaces.

**Pull request merged**